features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/life-cycles/template.jag

```diff
@@ -46,156 +46,56 @@ function setChecklistItemVal(element,order) {
-                $('#messageModal').modal({
-                    backdrop: 'static',
-                    keyboard: false
-                });
-                $(".modal-header .close").hide();
-                $(".modal-footer").html("");
-                $(".modal-title").html(i18n.t("Please wait"));
-                $(".modal-body").addClass("loadingButton");
-                $(".modal-body").css({
-                    "margin-left": 25
-                }); //todo param_string
-                $(".modal-body").html(i18n.t("Changing the API lifecycle status") + " : " + apiName + " - " + version);
-
-                jagg.post("/site/blocks/life-cycles/ajax/life-cycles.jag", {
-                    action: "updateStatus",
-                    provider: provider,
-                    name: apiName,
-                    version: version,
-                    status: status
-                }, function(result) {
-                    if (!result.error) {
-                        var jsonPayload = result.status.workflowResponse.jsonPayload;
-
-                        if (result.status.stateChangeStatus == 'REJECTED') {
-                            showWorkflowRejectedMessage();
-                        } else if (jsonPayload != null && jsonPayload != "") {
-                            var apiInfo = {};
-                            apiInfo.provider = provider;
-                            apiInfo.name = apiName;
-                            apiInfo.version = version;
-                            handleWorkflowRedirection(jsonPayload, apiInfo);
-                        } else {
-                            ///test
-                            $('#statusUpdateMsg').show();
-                            setTimeout(function() {
-                                location = ''
-                            }, 1000)
-                        }
-                    } else {
-                        if (result.message == "AuthenticateError") {
-                            jagg.showLogin();
-                        } else {
-                            jagg.message({
-                                content: result.message,
-                                type: "error"
-                            });
-                        }
-                    }
-                }, "json");
-            },
-            cancelCallback: function() {
-                $('#messageModal').modal('hide');
-                $(".modal-footer").html("");
-            }
-        });
+        messageContent = i18n.t("All the previous versions of the API will be deprecated. Continue?");
     } else {
-        jagg.post("/site/blocks/life-cycles/ajax/life-cycles.jag", {
+        var displayStatus = status.toLowerCase();
+        if (status == 'Demote to Created') {
+            displayStatus = displayStatus.replace("to created", ""); // Remove "to created" part
+        }
+        messageContent = i18n.t("Are you sure you want to ") + displayStatus + " this API?";
```

Comment:
making only a part of the message i18n is not useful. Shall we make the entire message i18n?

Reply:
In an early implementation,
messageContent = i18n.t("Once the API is deprecated; you can only retire the API as the next step.Are you sure you want to deprecate the API") + " - " + apiName + " - " + version + "?";
part of the message is taken to i18n and append variables to it.

Reply:
Complete the change as requested.